test/page-objects/login.ts

```diff
@@ -1,17 +1,20 @@
-import { login } from 'app/server/paths';
+import { login, dummyLogin } from 'app/server/paths';
 import { BasePage } from 'test/page-objects/base';
+const config = require('config');
+
+const enableDummyLogin: boolean = config.get('enableDummyLogin');
```

Comment:
Same thing here, we need the stupid check for `config.get('enableDummyLogin') === 'true'`

Reply:
Done